frontend/public/components/utils/datetime.ts

```diff
@@ -56,6 +56,14 @@ export const utcDateTimeFormatter = new Intl.DateTimeFormat(getLocale() || undef
   timeZoneName: 'short',
 });
 
+export const twelveHoursdateTimeNoYear = new Intl.DateTimeFormat(getLocale() || undefined, {
+  month: 'short',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+  hour12: true,
+});
+
```

Comment:
Can we make it a part of our utils folder?

Reply:
possible, let me do this